tests/integration/targets/rds_instance/roles/rds_instance/tasks/test_processor.yml

```diff
@@ -105,8 +105,10 @@
       - assert:
           that:
             - not result.changed
-            - 'result.pending_modified_values.processor_features.coreCount == "{{ modified_processor_features.coreCount }}"'
-            - 'result.pending_modified_values.processor_features.threadsPerCore == "{{ modified_processor_features.threadsPerCore }}"'
+            - 'result.pending_modified_values.processor_features.coreCount == "{{ modified_processor_features.coreCount }}"
+                or result.processor_features.coreCount == "{{ modified_processor_features.coreCount }}"'
```

Comment:
I don't understand this change. What is this testing?

Reply:
The modifications here are to ensure then idempotence check passes when the change in processor features are actually applied (as opposed to still pending). I had one or two tests fail here running locally

Reply:
That makes sense, thanks for explaining.

Reply:
of course!